CI/travis/build_double_down.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Should we put this into some/all of our docker images?  I don't know how long it takes to build this, but it would be great it we didn't add to the build time for DAGMC for every PR.

Reply:
Good point! I'll get that added to the image.

Reply:
You may want to confer with @bam241 about how best to add it to the matrix within Travis or wait until he gets CircleCI going?

Reply:
Indeed. @bam241 what do you think?

Reply:
Either way works for me.

it will probably be easier for you to wait for the CircleCI to be merge.
(It is almost ready. I think if it got some more reviews it could be merged today)